### PR TITLE
docs: create tutorial for task csv-to-cd (2)

### PR DESCRIPTION
**Pull request by degenaro**
## Types of changes

- [ ] Hot fix (emergency fix and release)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Documentation (change which affects the documentation site)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Release (`develop` -> `main`)

## Quality assurance (all should be covered).

- [ ] My code follows the code style of this project.
- [x] Documentation for my change is up to date?
- [ ] My PR meets testing requirements.
- [x] All new and existing tests passed.
- [x] All commits are signed-off.

## Key links:

- [Sonar coverage](https://sonarcloud.io/dashboard?id=compliance-trestle)

# Before you merge

- Ensure it is a 'squash commit' if not a release.
- Ensure CI is currently passing
- Check sonar. If you are working for a fork a maintainer will reach out, if required.

### CODE REVIEW

File: docs/tutorials/task.csv-to-oscal-cd/transformation.md [hunk 1]
```diff
@@ -0,0 +1,453 @@
...
+<td>A description of the collector of the actual state (evidence) associated with the policy (desired state) including the method (interview or examine or API) and questionaire
+<td>component.control-implementation.prop["Fetcher_Description"]
+<td>Fetch whether password policy requires minimum length of 12 characters
+
+<tr style="text-align:left;vertical-align:top">
+<td>Resource_Instance_Type
+<td>String
+<td>optional
+<td>A textual label that uniquely identifies the resource's type, to differentiate between resources. For example DB2 vs. Object Storage.
+<td>component.control-implementation.prop["Resource_Instance_Type"]
```
Comment (vikas-agarwal76) [inline docs/tutorials/task.csv-to-oscal-cd/transformation.md:158]: "A textual label that uniquely identifies a resource (component) type from the resource instance id. This text is part of all instance ids of a particular resource at runtime. For example the text 'db2' is part of all instance ids of resource DB2.

Comment (degenaro) [reply docs/tutorials/task.csv-to-oscal-cd/transformation.md:158]: Fixed.

**Comment by enikonovad** [comment]
@degenaro is this ready for the review?

**Comment by degenaro** [comment]
Yep.

**Review by enikonovad — APPROVED** [review]
LGTM

**Comment by degenaro** [comment]
Thx for the review!

**Pull request merged**